pkg/testing/integration_test.go

```diff
@@ -75,6 +78,13 @@ func newSecretProfile() *secretProfile {
 	}
 }
 
+var (
+	setUpOnce = sync.Once{}
+	setUpDone = make(chan struct{})
+)
+
+const controllerNamespace = "integration-test-controller"
```

Comment:
You could have controllerNamespace be a variable and tag a rand(6) on the end if you wish.  Just in case there are ever multiple jobs running integration test in parallel.

Or keep it this way.

Reply:
That would require a cleanup in Teardown suite with no easy way to know when really done.  This works as is.

Reply:
Oh -- it's wouldn't be that hard.  We could move namespace create and cleanup to function `Test` before and after the invocation of `TestingT()`.

That would have advantage of tearing down namespace and cleaning up after itself.

Your call.

Reply:
Deleting it would be difficult - I would need to add a wait group and keep track of the number of suites that have run. Here I just clean up before running.